src/main/java/seedu/dengue/model/overview/PostalOverview.java

```diff
@@ -52,13 +52,13 @@ public void update(List<Person> personList) {
     }
 
     private static String[] splitLocationName(String locationName) {
-        return locationName.split(", ");
+        return locationName.split(POSTAL_DELIMITER);
```

Comment:
If you are splitting by the "," it may be misleading to call it the delimiter as delimiter implies that you have reached the end of a line/file

Reply:
Thanks for catching that, but 'delimiter' is also used to refer to a character that splits up a string! e.g. in `'a.b.c'.split('.')` for Python, `'.'` is called the delimiter.
I think I'll keep it as is?